internal/repository/prisma/dbsqlc/sqlc.yaml

```diff
@@ -1,6 +1,8 @@
 version: "2"
 sql:
   - engine: "postgresql"
+    # database:
```

Comment:
comment?

Reply:
Leaving it for now as it was very helpful in addressing some strange `sqlc` behavior.